bin/ch/$262.js

```diff
@@ -0,0 +1,24 @@
+R"(
```

Comment:
yes, this is raw string in c++1x. not sure if the VS2012 support this. if not might need change this back to a messy byte array or load script file in running time